README.md

```diff
@@ -19,7 +19,7 @@ microservices, cloud native and container-based (Docker, Kubernetes, Mesos) arch
 
 ## Installation Requirements
 
-SkyWalking Python Agent requires SkyWalking 8.0+ and Python 3.5+.
+SkyWalking Python Agent requires SkyWalking 8.0+ and Python 3.6+.
```

Comment:
OK, since the requirement here is version-ed, users should check the corresponding doc of their versions, this PR is good to me now